### ISSUE TITLE
SignupUserResponse object incorrectly passes user id.

### ISSUE THREAD

**Issue by stephenstroud** [auth0/auth0.net#304]
Calling HTTP POST to - /dbconnections/signup

Returns 
```
{
 "user_id": "AUTH0|232424",
 "email": "someone@example.com",
 "email_verified" : false
}
```

https://github.com/auth0/auth0.net/blob/1543db7205f5df8e367121cb5b373a4ef85f7a72/src/Auth0.AuthenticationApi/AuthenticationApiClient.cs#L374

Trying to deserialize using:

https://github.com/auth0/auth0.net/blob/1543db7205f5df8e367121cb5b373a4ef85f7a72/src/Auth0.AuthenticationApi/Models/SignupUserResponse.cs#L26

API authentication doc shows a different response object

https://auth0.com/docs/api/authentication?http#signup

Is this an API version issue?

Thanks

**Comment by damieng** [comment]
It's possible the API has moved on from the original definition although I'm not sure what happened to the `_id` name.

**Comment by damieng** [comment]
It looks like the API actually changed back in 2016 but nobody has noticed it is broken here, in the docs and in our Java API (our other management APIs return dynamic types).

**Comment by stephenstroud** [comment]
@damieng Will the .Net client be updated soon to reflect this issue?

**Comment by damieng** [comment]
I've been doing some more investigating and am able to get `_id` back... I'm working with internal teams to figure out why you can get different id results and under what circumstances.

**Comment by stephenstroud** [comment]
@damieng Any update?

**Comment by damieng** [comment]
I haven't had cycles yet to determine what id is returned under what circumstances yet as I'm heads down on the 7.0.0 release. Will let you know if/when I have more information.

**Comment by damieng** [comment]
The rule is that _id is returned when using a standard connection and user_id when using a custom connection. We should add both properties to the deserialized response class with appropriate comments.

**Comment by stephenstroud** [comment]
Sounds good, I did the same deserialised tactic with a null check in a wrapper class.

**Comment by damieng** [comment]
Given that the upcoming 7.0.0 release has breaking changes and will be in beta for a week we've gone ahead and published 6.5.5 that includes this fix.  It should be available no NuGet now.

**Comment by stephenstroud** [comment]
awesome
